tests/test_multi_standbys.py

```diff
@@ -296,3 +296,67 @@ def test_020_start_node3_again():
     assert node1.has_needed_replication_slots()
     assert node2.has_needed_replication_slots()
     assert node3.has_needed_replication_slots()
+
+def test_021_most_advanced_standby_inaccessible():
+
+    global node1
+    global node2
+    global node3
+
+    node1.set_candidate_priority(50)   # the current primary
+    node2.set_candidate_priority(100)  # the next primary
+    node3.set_candidate_priority(0)    # never candidate for a failover
```

Comment:
Consider using `assert` or `eq_` for the priorities that are already set the way you need them to be, so that we don't have to wait for a full round-trip with the monitor and the primary 3 times here.

Reply:
Also please set the candidate priorities for nodes in the expected order (node1, node2, node3), making it easier to follow along.

Reply:
> Also please set the candidate priorities for nodes in the expected order (node1, node2, node3), making it easier to follow along.

Aren't they  already in that order? Or do you mean the highest should be the first line?